app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -154,7 +169,9 @@ def images
   end
 
   def volumes
-    collector.volumes.each do |vol|
+    collector.volumes.each do |volume_ref|
```

Comment:
Seems like these changes to the volume parser are unintentional? I think you might need to rebase to include #257

Reply:
not sure how it got through. I may rebase it again and see.

Reply:
@Kuldip-Nanda these changes to the volume parsing still need to be removed. After that and removing the `[WIP]` from the PR title I think this is ready to merge!